e2e/scenario/fixture/api.spec.js

```diff
@@ -85,14 +85,14 @@ describe('api', function() {
       expect(err.status).toBe(403);
     });
 
-    request.csrf({
-      url: restServiceMockUrl + '/form'
-    }).then(function () {
+    request.authenticate(new CsrfAuthenticator({
+      csrfFetchUrl: restServiceMockUrl + '/form'
+    })).then(function () {
       request.post(restServiceMockUrl + '/form').send({
```

Comment:
subsequent calls should be scheduled on the flow, no need to calling in promise resolve callback